Makefile

```diff
@@ -0,0 +1,94 @@
+lint:
+	go fmt ./...
+	go vet ./...
```

Comment:
go lint is not `fmt` & `veŧ` commands, it's more something like https://github.com/golangci/golangci-lint (which are all nice to have).
I think we could have a script in `./script` running lint, and a Makefile target it to execute it.

Reply:
I've got another series locally to add golangci-lint and resolve the errors. I'll push that once we get this in.